composer.json

```diff
@@ -9,22 +9,21 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.0 || ^8.1",
```

Comment:
Drop php 8.0 support

Reply:
@mpysiak We discussed that topic. What we think we need to support 8.0 version for sylius 1.12 as it's core of that version.

Reply:
PHP 8.0 is not supported. Sylius 1.12 will work on PHP 8.1

Reply:
@senghe said that we need to support PHP 8.0. Maybe you can discuss it.

Reply:
Yup, Sylius 1.12 supports PHP 8.0, so we decided to keep it, as some of projects potentially could use 8.0 with it.